### ISSUE TITLE
Adjust width of the chechbox area in the food list

### ISSUE THREAD

**Issue by thomas-esau** [davidhealey/waistline#802]
I struggle a little bit with the same problem as the author of issue https://github.com/davidhealey/waistline/issues/439 - the area to tap, that activates the checkbox is too slim for me. I have another proposal for a solution though: It would be nice, if the checkbox activation area had about the same width as the button for new entries in the bottom right corner. If this is too big of a change for the UI, maybe it could just be a toggleable option under Settings -> Appearance.

**Comment by EmilJunker** [comment]
I think increasing the width of the checkbox a bit would indeed help, and it should be relatively easy to do.

@thomas-esau I invite you to try and implement this change yourself and make a pull request. The field of contributors to this app has become sparse and any help is welcome. You have a much better chance of getting this change shipped if you do it yourself rather than wait until someone else comes around to do it.

There are build instructions in the contributors guide: https://github.com/davidhealey/waistline/blob/master/CONTRIBUTING.md#code-contribution

**Comment by thomas-esau** [comment]
Yes sure, I could give it a try. I've never worked with Cordova or Framework7, but since these are frameworks for the usual webdev stack, it should still be doable for me.
